### PR TITLE
Update default start page to included startpage/index.html

### LINKED ISSUE #312: Create starting page to show as default page when launching new Edge target

### PR DESCRIPTION
**Pull request by bgoddar**
This PR updates the default page used when launching Microsoft Edge from about:blank to the startpage/index.html included in the repo

Closes #312

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -564,6 +565,7 @@
         "@types/ws": "7.2.6",
         "@typescript-eslint/eslint-plugin": "4.19.0",
         "@typescript-eslint/parser": "4.19.0",
+        "copy-webpack-plugin": "^8.1.1",
```
Comment (vidorteg) [inline package.json:568]: let's keep a fixed version. (no ^ [dont know what is the name for this symbol :D] )

File: package.json [hunk 2]
```diff
@@ -564,6 +565,7 @@
         "@types/ws": "7.2.6",
         "@typescript-eslint/eslint-plugin": "4.19.0",
         "@typescript-eslint/parser": "4.19.0",
+        "copy-webpack-plugin": "^8.1.1",
         "eslint": "7.22.0",
         "eslint-plugin-import": "^2.22.1",
```
Comment (vidorteg) [inline package.json:570]: could take out the other & in eslint-plugin (probably fell through the cracks)

File: webpack.config.ts [hunk 3]
```diff
@@ -1,51 +1,61 @@
...
-            devtoolModuleFilenameTemplate: '../[resource-path]',
-            filename: '[name].js',
-            libraryTarget: 'commonjs2',
-            path: path.resolve(__dirname, 'out'),
-        },
-        stats: 'errors-only', // Bug ws package includes dev-dependencies which webpack will report as warnings
-        target: 'node',
-    },
-];
+import copyPlugin from 'copy-webpack-plugin';
```
Comment (vidorteg) [inline webpack.config.ts:1]: changes LGTM but do we know why everything is marked as new?

**Review by vidorteg — APPROVED** [review]
LGTM % nit.

File: webpack.config.ts (hunk 3 above)
Comment (bgoddar) [reply webpack.config.ts:1]: Olivia's PR that modified all double quotes to single just landed. Merge conflicts everywhere. I just rebased. Much easier to review

File: package.json (hunk 1 above)
Comment (bgoddar) [reply package.json:568]: carot :)

Comment (bgoddar) [reply package.json]: Done

**Pull request merged**